packages/flutter_tools/lib/src/commands/build_apk.dart

```diff
@@ -221,8 +221,8 @@ class BuildApkCommand extends BuildSubCommand {
   }
 
   @override
-  Future<int> runInProject() async {
-    await super.runInProject();
+  Future<int> runCmd() async {
```

Comment:
Just a style nit: we prefer to use full words in variable name.  So, this might be "runCommand"